.github/dependabot.yml

```diff
@@ -1,14 +1,24 @@
 version: 2
 updates:
   - package-ecosystem: "gomod"
-    directory: "/"
+    directory: "/provider"
     schedule:
       interval: "daily"
       time: "08:00"
     labels:
       - "dependencies"
     commit-message:
-      prefix: "feat"
+      prefix: "chore"
+      include: "scope"
+  - package-ecosystem: "gomod"
+    directory: "/sdk"
+    schedule:
+      interval: "daily"
```

Comment:
If you don't want to be overloaded with PRs for every Pulumi release, I would lower the interval to monthly.

Reply:
Good point! I changed it to `weekly` on a `Sunday`. I am afraid to miss a patch of a critical vulnability.